### DISCUSSION TITLE
requests to join GitHub student denied

### DISCUSSION THREAD

**Discussion by maisa1n** [community/community#33763] [GitHub Education] [answered]
![Screenshot_20220922-070048_Chrome](https://user-images.githubusercontent.com/113845428/191718689-e3e6f03e-f689-43c1-8aaf-e83c5b73af7e.jpg)

hello, i'm from brazil, so if this have english mistakes i'm sorry. so I tried many times to be joining in the GitHub student, would it be possible to talk to the platform, for them to try to help me? cuz i got the document confirming that i study at this school, this should be enough, isn't it? 

**_I GOT IT!!_**

So I finally managed, after they deny 7 times hahahaha, I'm loving the student github is wonderful, here goes as I managed: In the first attempts I was putting only the image of the registration statement, but on this last attempt I took a picture of me holding the statement next to my face, second: my location was not active, and my location was not allowed in the browsing settings (google), lastly the name of my school, in particular, is written in capital letters and before I was putting in lowercase letters.

Well, I hope this can help someone is going through the same problem as mine, and I also appreciate the people in the comments that helped me too.

PT-BR:
Então eu finalmente consegui, depois que eles negaram 7 vezes hahahaha, estou amando o programa de estudante do github é maravilhoso. e aqui vai como eu consegui ser aprovada: Nas primeiras tentativas eu estava colocando apenas a imagem da declaração de matrícula, mas nessa última (e bem-sucedida) tentativa eu tirei uma foto minha segurando a declaração ao lado do meu rosto, segundo: minha localização não estava ativa no celular, e minha localização não estava permitida nas configurações do navegador (google), por último, o nome da minha escola, em especifico, é escrito em letras maiúsculas e antes eu estava colocando letras minúsculas. Essas coisas provavelmente estavam me impedindo de ser aprovada.

Espero que isso possa ajudar alguém que esteja passando pelo mesmo problema que o meu, e também agradece as pessoas nos comentários que me ajudaram!

**Comment by samuel-ma** [comment]
@maisa1n 
You can always go to [Github Support](https://support.github.com/) to find solutions to similar problems
The Github student pack is a very common problem, you should find an answer for your problem relatively quickly once you've gone through [Github Support](https://support.github.com/)

**Comment by SilasRodrigues19** [comment]
@maisa1n Se você tiver e-mail institucional, adiciona ele no seu GitHub nesse [link](https://github.com/settings/emails). O meu foi recusado várias vezes e só deu certo depois que eu adicionei o e-mail institucional, pode dar certo com você também

**Reply by maisa1n** [reply]
eu já coloquei 😥 mesmo assim não deu certo, o nome da instituição que vc estuda estava listado lá? pq o meu não aparece, e o documento que comprova a matrícula pode ser em português mesmo né?

**Reply by SilasRodrigues19** [reply]
Eu lembro que tinha colocado o nome da instituição mas não tinha nada listado e enviei tudo em português mesmo. No meu caso demorou porque dava problema com a foto que comprovava a matrícula, tive que cancelar e tentar umas 3 vezes pra dar certo

**Reply by maisa1n** [reply]
no meu não aparece a o nome da escola, daí eu coloquei o nome da rua mesmo... oq você fez no seu, já que o nome da instituição não apareceu

**Reply by SilasRodrigues19** [reply]
Eu só coloquei o nome mesmo que é Fatec, mas ai como eu tinha o email com @fatec acho que ajudou no processo e no documento que envia pra confirmar que é estudante eu mandei um print do sistema com a grade e o status de cursando, mostrando o semestre, as matérias, etc

**Comment by ghost** [comment]
@maisa1n 
 I  was face this problem and I went to the website to translate the image and make it and I uploaded it and   they accepted me

**Reply by maisa1n** [reply]
So what did you translate? the registration document?

**Reply by ghost** [reply]
@maisa1n    my school ID

**Comment by samus-aran** [comment, marked as answer]
Hi @maisa1n. :wave: The verification process for students is pretty robust but it should have described an error or rejection notice. Can you share the rejection notice(s) when you applied? Perhaps that will provide us an indication as to why it was rejected and help you adjust accordingly to have a successful application.

**Reply by maisa1n** [reply]
hiii, I added the image with the rejection notice

**Reply by samus-aran** [reply]
Thanks! 

Okay let’s break this down and review what it says.

Are you using a Photo ID with current enrollment date? If so, is it in gregorian calendar format (dd/mm/yyyy)?
Had you modified the Photo proof i.e. used a software to hide personal details?
Is the school you are registering with selectable on the GitHub application form?
On the registration certificate what does it contain? Does it contain dates, the name of the school?

**Reply by maisa1n** [reply]
no, I used an enrollment statement. yes, there is a date on the document. i don't change anything, I just took a photo of the document printed by the school. when I put the name of my school it does not appear in the list of schools. and yes, in the declaration it has the name of the school, the date and it is also signed by the school secretary

**Reply by samus-aran** [reply]
Okay :)
Hmmm, since you're posting in Portuguese I would suggest you *ensure* that the school name on your proof matches exactly the name you are selecting/creating since it's a new school (in the verification). Auto translation is awesome but little variations can have an impact on the process. Plus having high res of the proof would help with identifying the lettering.

**Reply by maisa1n** [reply]
ok, thank you so much! I will keep trying.

**Reply by Sjskan** [reply]
> ok, thank you so much! I will keep trying.

**Comment by harpercammer** [comment]
If you're in Acellus Academy, GitHub makes it impossible for you to sign up for a student account. They streamline the process for affluent students from traditional backgrounds to sign up for free accounts, but filter everyone else out.

**Reply by maisa1n** [reply]
hii, i got in, i send a pic of me holding the registration statement and it worked 😊

**Reply by harpercammer** [reply]
Where were you able to send the statement to? I created a ticket but haven't heard anything back yet.

**Reply by harpercammer** [reply]
<img width="1079" alt="Screen Shot 2022-09-28 at 12 34 29 PM" src="https://user-images.githubusercontent.com/114508078/192862038-3338c239-0aec-4d3d-9114-6e124195479d.png">

I can't do anything. It refuses to allow me to even submit anything.

**Reply by samus-aran** [reply]
Hey @harpercammer. I'm sorry to hear you feel that way but here at GitHub we do support many types of education institutions including Acellus Academy. Let's see if we can assist so your application goes through (which I would assume is still being processed currently). Based on the above image it does ask for the application to include a dated school ID because this type of evidence has been a good verifier for the school you are attending, do you have this? 
If not, then consider what evidence you do have to indicate you are: 
1. who you say you are,
2. that you are a student 
3. that you attend that school (name of school)
4. relevant dates (DD/MM/YYYY) (multiple formats)

I see in your image (a report card) some of these are true, but not all. Perhaps if this application is rejected (hopefully it won't) that you can re-apply with more identifying evidence e.g. Student ID or a transcript with hitting all the above key points.

**Reply by ghost** [reply]
hy guys i cant send request always ask mefor modify my location and he didnt send me message

**Reply by ghost** [reply]
![Uploading Screenshot_2022-09-30-03-25-32-872_com.android.chrome.jpg…]()

Anyone can help

**Reply by maisa1n** [reply]
the photo is not loading, you can send the photo again, so I can see if I can help you

**Reply by maisa1n** [reply]
hi @harpercammer You got it? maybe you can print this report card and do like i did, send a picture of you holding the report card next to your face, and mark it as school id

**Comment by fares-seghir** [comment]
hi so i tried to apply for student campus github but when i applied i got an error message where they told me to add location property and when i I added it and redid my request they told me that my photo had already been used so I can't use it anymore and I don't have another picture
![image](https://user-images.githubusercontent.com/115456434/194827959-77d33cd4-7e1e-4c7b-8c84-ae5d5027d050.png)

**Comment by Abdou-MTR** [comment]
i couldn't  get my github student even when i added my student ID card and i got this problem
![Screenshot 2022-10-13 185427](https://user-images.githubusercontent.com/113215935/195670571-8ee249d0-ccaf-46fe-a022-2f780e8644ac.png)
![Student ID Card](https://user-images.githubusercontent.com/113215935/195670718-2e96eb0e-957e-453e-94ef-6efda5552d63.jpg)

this is my  student ID card
i hope you reverify it

**Reply by maisa1n** [reply]
try follow the instructions written in the image... 
1. complete profile on github 
2. take a picture of YOU with the id card next to your face
3. try to put the translation of the id card in the photo too 

These things worked for me, maybe they will work for you too. keep trying  😉

**Reply by Abdou-MTR** [reply]
thank you

**Reply by Undertone0809** [reply]
Actually, I have been tried it several times but still  saying "The image not appear to contain my school name". I don't know why it doesn't work by so many times. I have already upload my school id card.

**Comment by GasComIT** [comment]
I am here to ask a question....
I have applied several times for github students but still cant get verified... 
I have all they document needed, i tried with different documents but all time they say, missing date , sometimes document size, and my last error was weird, they said document must be one of ( school id or transcript {dated}) - i uploaded but still get rejects... First I hate AI verifications because they spot mistakes that arent there... Like failing to see dates on my documents.. What should i do ??

**Reply by Undertone0809** [reply]
I have the same problem. But my friend can upload the file instead of taking picture. I can not find it...
